# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1325484867 28800
#      Sun Jan 01 22:14:27 2012 -0800
# Branch stable
# Node ID 2e68a7a1721a75f577631beb771cb260bd40ee2d
# Parent  84f38aeb9f0e9683e03bdd21f80072849cba8394
doc: Report Octave version when broadcasting semantics changed in start up WARNING.

version.h.in: Include version number in WARNING message on startup.

diff --git a/src/version.h.in b/src/version.h.in
--- a/src/version.h.in
+++ b/src/version.h.in
@@ -83,12 +83,12 @@ For more information, visit http://www.o
 
 #define OCTAVE_NAME_VERSION_COPYRIGHT_COPYING_WARRANTY_AND_BUGS \
   X_OCTAVE_NAME_VERSION_COPYRIGHT_COPYING_WARRANTY_AND_BUGS ("")
 
 #define OCTAVE_STARTUP_MESSAGE \
   X_OCTAVE_NAME_VERSION_COPYRIGHT_COPYING_WARRANTY_AND_BUGS \
     ("  For details, type `warranty'.") "\n\n" \
   "For information about changes from previous versions, type `news'.\n\n\
-** WARNING ** Semantics for arithmetic operators have changed and now\n\
-              broadcast. Type `doc broadcast' to get a description of\n\
-              the changes."
+** WARNING ** Semantics for arithmetic operators were changed\n\
+              in version 3.6.0 and now broadcast.\n\
+              Type `doc broadcast' to get a description of the changes."
 #endif
